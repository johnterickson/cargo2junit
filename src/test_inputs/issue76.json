{ "type": "suite", "event": "started", "test_count": 1 }
{ "type": "test", "event": "started", "name": "test::set_up_subscriber" }
{"timestamp":"2023-04-02T17:20:52.256633Z","level":"INFO","fields":{"message":"Subscriber Initialised"},"target":"repro72::test"}
{ "type": "test", "name": "test::set_up_subscriber", "event": "ok", "exec_time": 0.0072669 }
{ "type": "suite", "event": "ok", "passed": 1, "failed": 0, "ignored": 0, "measured": 0, "filtered_out": 0, "exec_time": 0.0079406 }
