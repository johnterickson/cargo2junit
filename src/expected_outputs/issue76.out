<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite id="0" name="cargo test #0" package="testsuite/cargo test #0" tests="1" errors="0" failures="0" hostname="localhost" timestamp="2023-04-02T23:45:27.211449886Z" time="0.0072669">
    <testcase name="set_up_subscriber" time="0.0072669" classname="test" />
  </testsuite>
</testsuites>
